internal/analytics/client.go

```diff
@@ -106,7 +107,8 @@ func Init(opts ...Option) error {
 func New(opts ...Option) *Client {
 	c := &Client{
 		version:    VersionInfo{},
-		userId:     GetUserId(),
+		userId:     GetUserId().String(),
+		cookieId:   GetUserId(),
```

Comment:
since we're using "User ID" as cookie id and eventually db id as user id, I would rename `GetUserId` to be `GetCookieId`

Reply:
Although it does become a user id if no database id is set (for example pg < 10)